### PR TITLE
Add index.js to components and pages folder and connect through files

### LINKED ISSUE #46: Add index.js to the components folder
The index.js should contain imports for all other components. 

This would allow us to just import everything from index.js

### LINKED ISSUE #113: Add index.js to the pages folder

### PR DESCRIPTION
**Pull request by Stephanie-A**
closes #113 closes #46

### CODE REVIEW

File: client/src/pages/ParentPortal.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import React, { Component } from 'react';
-import PrimaryNavBar from './../components/Navbar';
-import StudentCard from './../components/StudentCard';
-import AnnouncementList from './../components/Announcements';
+import { PrimaryNavBar, StudentCard, AnnouncementList } from './../components';
```
Comment (Samyakkumar) [inline client/src/pages/ParentPortal.js]: I think the import name here should be PrimaryNavbar or change the component use on line 36 to PrimaryNavBar.

It gives an error while rendering right now

Comment (Stephanie-A) [reply client/src/pages/ParentPortal.js]: Wait it doesn't give me an issue when I render it

Comment (Samyakkumar) [reply client/src/pages/ParentPortal.js]: Oh damn it. I meant to comment in the PaymentPage.js file. That page has a <PrimaryNavbar /> but import PrimaryNavBar

File: client/src/pages/PaymentPage.js [hunk 2]
```diff
@@ -1,10 +1,7 @@
 import React, { Component } from 'react';
-import PrimaryNavbar from './../components/Navbar';
+import { PrimaryNavBar, PaymentPageContent, PaymentPageProgress } from './../components';
```
Comment (Samyakkumar) [inline client/src/pages/PaymentPage.js]: Here. It says PrimaryNavBar. But on line 36 we use PrimaryNavbar

Comment (Stephanie-A) [reply client/src/pages/PaymentPage.js]: It's changed to PrimaryNavBar in my code...

Comment (Samyakkumar) [reply client/src/pages/PaymentPage.js]: Wait did you push all the changes then? Because opening the file on github shows what I'm seeing.

Comment (Stephanie-A) [reply client/src/pages/PaymentPage.js]: That's so weird, I thought I pushed it, but I'll fix it.

**Pull request merged**